src/amber/cli/templates/scaffold/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -63,4 +63,14 @@ class <%= @name.capitalize %>Controller < ApplicationController
     end
     redirect_to "/<%= @name %>s"
   end
+
+  def <%= @name %>_params
+    params.validation do
+      <%- @fields.each do |field| -%>
+      <%- unless %(created_at updated_at).includes? field.name -%>
+      required(:<%= field.name %>) { |p| p.to_s.empty? }
```

Comment:
This is failing for checkboxes that may not send a value in the params.

```bash
amber g scaffold post title:string body:text published:bool
```

When trying to create a new blog post, I get an error:
```
ERROR: Validation failed. [Amber::Validators::Error(@param="title", @value="This is a test", @message="Field title is required"), Amber::Validators::Error(@param="body", @value="Testing", @message="Field body is required"), Amber::Validators::Error(@param="published", @value="0", @message="Field published is required")] (Amber::Exceptions::Validator::ValidationFailed)
```